modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -1006,6 +1036,55 @@ private ServiceContext _createServiceContext(
 		return serviceContext;
 	}
 
+	private void _disassociateRelatedModel(
+			DTOConverterContext dtoConverterContext,
+			ObjectDefinition objectDefinition,
+			ObjectRelationship objectRelationship, long primaryKey,
+			ObjectDefinition relatedObjectDefinition)
+		throws Exception {
+
+		if (!relatedObjectDefinition.isUnmodifiableSystemObject()) {
```

Comment:
I think it is more clear in this case, if we have `if` and `else` to have a possitive clause